holidays/countries/cape_verde.py

```
@@ -0,0 +1,191 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)